install/kubernetes/helm/istio/values.yaml

```diff
@@ -81,6 +81,8 @@ tracing:
 #
 kiali:
   enabled: false
+  dashboard: 
+    grafana: {}
```

Comment:
This might be confusing to people because, IIUC, this is actually the external_services.grafana.auth section (as it is known in the kiali configmap). I don't know if people care, but if someone sees this "dashboard.grafana" dict, they are probably going to think its that kiali dict `external_services.grafana` and not limited to the auth section.

Perhaps call this `grafanaAuth` or something more specific so it is clear.

Reply:
@jmazzitelli updated as per your comments.